tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "extends": "eslint-config-upleveled/tsconfig.base.json"
```

Comment:
do we need `"includes"` here? I think the default doesn't have the includes we want, right?